traits/observers/expression.py

```diff
@@ -10,9 +10,18 @@
 
 import functools as _functools
 
+from traits.observers._dict_item_observer import (
+    DictItemObserver as _DictItemObserver,
+)
+from traits.observers._list_item_observer import (
+    ListItemObserver as _ListItemObserver,
+)
 from traits.observers._named_trait_observer import (
     NamedTraitObserver as _NamedTraitObserver,
 )
+from traits.observers._set_item_observer import (
+    SetItemObserver as _SetItemObserver,
+)
 from traits.observers._observer_graph import (
     ObserverGraph as _ObserverGraph,
 )
```

Comment:
Now I am less a fan of aliasing because we are going to have an `api.py` and this is a bit of nuisance. The top-level functions in this module will be exposed in the `api.py`. I am collecting this sort inconsequential renaming at the end.

Reply:
Now that `api.py` in master, this PR should expose the new expression top-level function there too.

Reply:
Yes, my preference would be not to do this aliasing. We don't bother with it elsewhere in Traits, and I don't think we've ever found a reason to.